src/ari_sxn_simbeamline/xrt_sim.py

```diff
@@ -0,0 +1,131 @@
+import sys
+
+# sys.path.append('/Users/jieminli/PycharmProjects/Beamline_sim/')
```

Comment:
please remove any commented out code, comments (#) should be left for comments only